src/components/stimuli/Stimuli.tsx

```diff
@@ -12,9 +17,16 @@ const generateCards = (stimuliUrls: Array<string>): Array<JSX.Element> => {
   return cards;
 };
 
+const DEFAULT_FILE_INPUT_TEXT = 'Choose an image...';
+
 const Stimuli: React.FC = () => {
   const [hasError, setErrors] = useState(false);
   const [stimuliUrls, setStimuliUrls] = useState([]);
+  const [isUploading, setUploading] = useState(false);
+  const [showToast, setShowToast] = useState(false);
+  const [responseMessage, setResponseMessage] = useState('');
+  const [file, setFile] = useState<string | Blob>('');
+  const [fileInputText, setFileInputText] = useState(DEFAULT_FILE_INPUT_TEXT);
```

Comment:
We probably don't need state for input text. See Experiment component for how to do this. The file state should probably be renamed to `selecteFile` for consistency and it's better to use a file object instead of a blob or string.